src/components/templates/ExperimentalSpace/ExperimentalSpace.tsx

```diff
@@ -200,7 +195,9 @@ const _ExperimentalSpace: React.FC<ExperimentalSpaceProps> = ({
         {!inHuddle && <p onClick={connectCallback}>Connect</p>}
         {inHuddle && <p onClick={shareScreenCallback}>Share screen</p>}
       </div>
-      <Dropdown title={projectTitle}>{trackOptions}</Dropdown>
+      <Dropdown options={[]} title="Choose a track to project">
```

Comment:
maybe we can make options default then? and add a default value in the Dropdown component?

Reply:
In my opinion, there is no need for it. This particular case is not going to show anywhere, this experimental space was developed by Colin for testing purposes. I didn't really know what the trackOptions are, so I just added empty array to avoid the compile error for the component. But it should definitely always accept options prop, which should be required.

Reply:
ok, can we use an ALWAYS_EMPTY_ARRAY then not to create a new object on each rerender?